meta/main.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
 - role: openmicroscopy.omero-python-deps
 - role: openmicroscopy.redis
   when: omero_web_runtime_redis
+- role: openmicroscopy.selinux-utils
```

Comment:
Is the convention to install by alphabetical order?
I think that will be more natural to have the "default" ones first then the optional